tests/attr/test_feature_ablation.py

```diff
@@ -345,15 +349,19 @@ def _multi_input_batch_scalar_ablation_assert(self, func):
 
     def _ablation_test_assert(
         self,
-        model,
-        test_input,
-        expected_ablation,
-        feature_mask=None,
-        additional_input=None,
-        ablations_per_eval=(1,),
-        baselines=None,
-        target=0,
-    ):
+        model: Module,
+        test_input: TensorOrTupleOfTensors,
+        expected_ablation: Tuple,
```

Comment:
This should include List[List[float]] and Tuple of these (to be more specific).

Reply:
revised to Union[List[List[float]], Tuple[List[List[float]]]],